packages/preset-tailwind/src/index.ts

```diff
@@ -281,25 +282,25 @@ export const fontSizes = [
   '4.5rem',
 ]
 
-export const baseFontWeights = {
-  hairline: '100',
-  thin: '200',
-  light: '300',
-  normal: '400',
-  medium: '500',
-  semibold: '600',
-  bold: '700',
-  extrabold: '800',
-  black: '900',
+export const baseFontWeights: Theme['fontWeights'] = {
+  hairline: 100,
+  thin: 200,
+  light: 300,
+  normal: 400,
+  medium: 500,
+  semibold: 600,
+  bold: 700,
+  extrabold: 800,
+  black: 900,
 }
 
-export const fontWeights = {
+export const fontWeights: Theme['fontWeights'] = {
```

Comment:
Similar comment as above. I'd drop fontWeights and baseFontWeights annotations.

Reply:
We dont have to annotate them - i did it thinking of it as a tutorial to build your own themes, it will catch typing issues earlier.
I can remove them - wdyt?

Reply:
We'll export `makeTheme` for TS users and preset maintainers. I thought about it for a longer time, and I think there`s little reason not to include it. Let's keep this open for a bit, I'll show you what I mean sometime on Discord, WDYT?

Reply:
sounds good